.github/workflows/release-production.yml

```diff
@@ -11,6 +11,7 @@ env:
 
 permissions:
   contents: read
+  packages: write
```

Comment:
Q: Shouldn't this be the only change?
Not sure why there are version changes of dependencies included in the PR also.
Can those be removed?